ui/app/AppLayouts/Communities/popups/TransferOwnershipPopup.qml

```diff
@@ -1,93 +1,153 @@
-            }
-            input.rightComponent: StatusButton {
-                anchors.right: parent.right
-                anchors.rightMargin: Style.current.halfPadding
-                anchors.verticalCenter: parent.verticalCenter
-                borderColor: Theme.palette.primaryColor1
-                size: StatusBaseButton.Size.Tiny
-                text: qsTr("Copy")
-                objectName: "copyCommunityPrivateKeyButton"
-                onClicked: {
-                    text = qsTr("Copied")
-                    Utils.copyToClipboard(root.privateKey)
-                }
-            }
+        CustomText {
+            text: qsTr("1. Send the %1 Owner token (%2) to the new owner’s address").arg(root.communityName).arg(token.name)
         }
 
-        StatusBaseText {
+        CustomText {
+            text: qsTr("2. Ask the new owner to setup the control node for %1 on their desktop device").arg(root.communityName)
+        }
+
+        StatusMenuSeparator {
             Layout.fillWidth: true
+        }
 
-            text: qsTr("You should keep it safe and only share it with people you trust to take ownership of your community")
-            wrapMode: Text.WordWrap
-            font.pixelSize: 13
-            color: Theme.palette.baseColor1
+        CustomText {
+            text: qsTr("I acknowledge that...")
         }
 
-        StatusBaseText {
-            Layout.fillWidth: true
+        StatusCheckBox {
+            id: ackCheckBox
 
-            text: qsTr("You can also use this key to import your community on another device")
-            wrapMode: Text.WordWrap
-            font.pixelSize: 13
-            color: Theme.palette.baseColor1
+            Layout.topMargin: -Style.current.halfPadding
+            Layout.bottomMargin: Style.current.halfPadding
+
+            font.pixelSize: Style.current.primaryTextFontSize
+            text: qsTr("My ownership rights will be removed and transferred to the recipient")
         }
     }
 
+    header: StatusDialogHeader {
+        headline.title: qsTr("Transfer ownership of %1").arg(root.communityName)
+        leftComponent: Rectangle {
+            height: 40
+            width: height
+            radius: height/2
+            color:Theme.palette.baseColor2
+
+            Image {
+                id: image
+
+                source: root.communityLogo
+                anchors.fill: parent
+                fillMode: Image.PreserveAspectFit
+                visible: false
+            }
+
+            OpacityMask {
+                anchors.fill: image
+                source: image
+                maskSource: parent
+            }
+        }
+        actions.closeButton.onClicked: root.close()
+    }
+
     footer: StatusDialogFooter {
-        leftButtons: ObjectModel {
-            StatusBackButton {
+        spacing: Style.current.padding
+        rightButtons: ObjectModel {
+            StatusButton {
```

Comment:
`StatusFlatButton` works better here, ie you don't have to specify/override the colors